apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -288,6 +292,12 @@
     <div class="ui divider hidden"></div>
 <% } %>
 
+<% if (isAdminSessionAdvisoryControlEnabledInTenant == true) { %>
+    <div class="ui warning message" data-componentid="login-page-admin-session-advisory-banner">
+        <%=adminSessionAdvisoryControlDescriptionOfTenant%>
```

Comment:
Just asking out of curiosity, will this content be i18n compatible?

Reply:
Thank you for bringing this up. According my knowledge, it is not. I will update the PR with the necessary changes.

Reply:
Since this is a user editable field, there is no way for us to make it i18n compatible.

Reply:
Created the issue https://github.com/wso2/product-is/issues/15630 to address this in a separate effort.

Reply:
Thanks for taking time to check this.

> Created the issue [wso2/product-is#15630](https://github.com/wso2/product-is/issues/15630) to address this in a separate effort.

+1